playbooks/runcible.yml

```diff
@@ -0,0 +1,104 @@
+---
+- hosts: all
+  become: true
+  roles:
+    - puppet_repositories
+    - pulp_repositories
+    - epel_repositories
+
+- hosts: all
+  tasks:
+    - name: 'Add OSTree repo'
+      yum_repository:
+        name: atomic7-testing
+        description: "Atomic testing repository"
+        baseurl: http://cbs.centos.org/repos/atomic7-testing/x86_64/os/
+        gpgcheck: no
+        enabled: yes
+      become: true
+
+    - name: 'Install packages'
+      yum:
+        name: "{{ item }}"
+        state: "present"
+      become: true
+      with_items:
+        - ruby-devel
+        - rubygem-bundler
+        - libxml2-devel
+        - ruby
+        - rake
+        - gcc
+        - wget
+        - git
+        - nss
+        - puppet
+        - glib2
+        - ostree
+
+    - name: "Install RVM gpg key"
+      shell: "gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3"
+
+    - name: "Install RVM"
+      shell: "curl -sSL https://get.rvm.io | bash -s stable --ruby"
+
+    - name: "Install librarian-puppet"
+      gem:
+        name: librarian-puppet
+        state: present
+
+    - name: "Clone puppet-pulp"
+      git:
+        repo: https://github.com/katello/puppet-pulp.git
+        dest: "/home/vagrant/puppet-pulp"
+
+    - name: "Deploy Puppetfile"
+      copy:
+        content: "forge \"https://forgeapi.puppetlabs.com\"\nmetadata"
+        dest: "/home/vagrant/puppet-pulp/Puppetfile"
+
+    - name: "Install Puppet modules"
+      shell: "/home/vagrant/bin/librarian-puppet install"
+      args:
+        chdir: "/home/vagrant/puppet-pulp"
+      remote_user: "vagrant"
+
+    - name: "Link pulp puppet module"
```

Comment:
Is this still needed with the new Puppetfile w/ metadata?

Reply:
Without this link you run into --  Error while evaluating a Resource Statement, Could not find declared class pulp at /home/vagrant/test.pp:1:1